src/client/cli/cmd/version.cpp

```diff
@@ -66,5 +90,27 @@ QString cmd::Version::description() const
 
 mp::ParseCode cmd::Version::parse_args(mp::ArgParser* parser)
 {
-    return parser->commandParse(this);
+    QCommandLineOption formatOption("format",
+                                    "Output list in the requested format.\n"
+                                    "Valid formats are: table (default), json, csv and yaml",
```

Comment:
Like here - "table" is default, so let's make it so.

Reply:
Calling `multipass version --format` returns a parsing error despite there being a default value of `table` specified in the constructor for the option. From the specification:
```c++
QCommandLineOption::QCommandLineOption(const QString &name, const QString &description, const QString &valueName = QString(), const QString &defaultValue = QString())

name = "format"
description = "Output version information in the requested format.\nValid formats are: table (default), json, csv and yaml"
valueName = "format"
defaultValue = "table"
```
I have verified that the `format` value in the option `parser->value("format")` is in fact set to `table` when no parameter is supplied to the option. From searching around this seems to be an issue others have encountered as well.

Reply:
> Calling `multipass version --format` returns a parsing error despite there being a default value of `table` specified in the constructor for the option. From the specification:

That's correct, we do want this to be the default, and for a value to be required by the `--table` option.

I just meant that the option is explicitly configured to have `table` as default, but the code did not reflect that. Like shown above, the output without `--format` is different to that of `--format=table`, and that's unexpected.

Reply:
Got it, I believe the updates to the code above should now resolve the issue.